mne/beamformer/_dics.py

```diff
@@ -48,6 +48,12 @@ def make_dics(info, forward, csd, reg=0.05, label=None, pick_ori=None,
     reg : float
         The regularization to apply to the cross-spectral density before
         computing the inverse.
+    noise_csd : instance of CrossSpectralDensity | None
+        Noise cross-spectral density (CSD) matrices. If provided, whitening
+        will be done. The noise CSDs need to have been computed for the same
```

Comment:
``The noise CSD matrices need to be computed``

Reply:
My reasoning with the grammas was that as you are supplying the CSDs are a parameter to this function, they already have been computed.